ansible/roles/test/files/ptftests/advanced-reboot.py

```diff
@@ -591,8 +592,9 @@ def runTest(self):
                 except TimeoutError:
                     self.log("Data plane never stop")
                     no_routing_start = datetime.datetime.min
+                    upper_replies = [self.nr_vl_pkts]
 
-                if no_routing_start is not None:
+                if no_routing_start is not None and no_routing_start != datetime.datetime.min:
```

Comment:
is it possible to get no_routing_start equal to None?
I think we have two possibility here. Either no_routing_start is equal to datetime.datetime.min or some value.

Also I'd better don't use datetime.datetime.min value, I'd better use None, which is our default in this case.

Reply:
Using `None` based approach we have to insert additional logic as `no_routing_start` is used for time diff calculation in below code.
If this is strongly required we can change the logic like (and of course do not preset min value in exception handler)

```
                if no_routing_start is not None:
                    no_routing_stop, _ = self.check_forwarding_resume()
                else:
                    no_routing_stop = datetime.datetime.min
                    no_routing_start = datetime.datetime.min
```

let me know if the second approach looks better for you.

Reply:
Yes. The second approach looks better for me.

Reply:
done